### ISSUE TITLE
Caret hangs up when trying to open the preview window

### ISSUE THREAD

**Issue by purelygrey** [careteditor/caret#219]
I tried to pinpoint why it happens but I'm still not sure what exactly causes it.

**Steps to reproduce:** 

1.  Paste the following:

```javascript
new Vue({
  el: '#app',
  data: {
    title: 'Hello World!'
  },
  methods: {
    changeTitle: function(event) {
      this.title = event.target.value; // ...
                                       // ...
                                       // ...
    }
  }
});
```

3. Press **Ctrl+Shift+=** a couple of times to increase font size to the max
4. Resize the window so it looks kind of like that: 

![image](https://cloud.githubusercontent.com/assets/16882824/21966324/4b1cc3de-db82-11e6-9e61-f4e1f6a2123f.png)

5. Press **Ctrl+Shift+R** 

**Result:** 

Caret hangs up and eats roughly ~700mb of memory:

![image](https://cloud.githubusercontent.com/assets/16882824/21966254/445dd9b2-db81-11e6-9189-db40848c713f.png)

**Comment by astoilkov** [comment]
We have successfully replicated the problem and have started working on it. Thanks for reporting.

**Comment by astoilkov** [comment]
This is now fixed in the latest Caret 1.13.0 release.

Closing this issue. Thanks for providing such a detailed description of the problem.
